### DISCUSSION TITLE
My martin catalog is empty

### DISCUSSION THREAD

**Discussion by epam-vkerkhoff** [maplibre/martin#1135] [Q&A] [answered]
Hello, 

I have a PostGIS DB with some POIs. Martin (theoretically) uses views to populate tiles which are then used by digitransit-ui. 

However, when I go to my Martin url/catalog all I get is []

I have checked the SRIDs of the coordinate column (of the views) and also of the source table and all looks good, still, no success in getting any vector tiles from Martin. 

Any hints?

**Comment by tomeronen** [comment]
@epam-vkerkhoff im a beginner in martin, but will be happy to try to help. Are you getting any warning logs when you start martin? I had a couple of instances where these warning were very helpful

**Comment by epam-vkerkhoff** [comment]
Hi @tomeronen, I have now removed my config file, which seemed to be incorrect. Entities are discoverd, but now they are not rendered. I get errors like the ones below: 
```
[2024-01-16T15:08:13Z DEBUG actix_web::middleware::logger] Error in response: Error("can not parse \"5334.pbf\" to a u32")
[2024-01-16T15:08:13Z INFO  actix_web::middleware::logger] 10.0.2.4 "GET /public.poi_coords_e_bike_charging_stations/14/8829/5334.pbf HTTP/1.1" 404 33 "https://angermuende.bbnavi.de/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/120.0.0.0 Safari/537.36" 0.000048
[2024-01-16T15:08:13Z DEBUG actix_web::types::path] Failed during Path extractor deserialization. Request path: "/public.poi_coords_e_bike_rentals/14/8829/5334.pbf"
```

The entity is there, and there is also data in the URL. You can check https://tiles.bbnavi.de/catalog. 

Thanks,

**Reply by nyurik** [reply]
@epam-vkerkhoff you are using a very old way of calling Martin that was changed ~2 years ago (?).  There should be no `.pbf` at the end, and you might not need `public.` in front of the `poi_coords_e_bike_charging_stations` tile source

**Reply by nyurik** [reply]
P.S. You seem to also use a relatively old martin tile server - the catalog format has been updated a few releases ago.

**Comment by epam-vkerkhoff** [comment]
Hello, 

I'm trying to get a tile where I know for a fact there should be data, yet I get an 204 HTTP response code (with curl, without .pbf extension) 

Here is the result: 
```
volker_kerkhoff@EPESMALW0288 ~ % curl -v https://tiles.bbnavi.de/poi_coords_bike_repair_shops/17/70105/43050
*   Trying 3.75.56.105:443...
* Connected to tiles.bbnavi.de (3.75.56.105) port 443
* ALPN: curl offers h2,http/1.1
* (304) (OUT), TLS handshake, Client hello (1):
*  CAfile: /etc/ssl/cert.pem
*  CApath: none
* (304) (IN), TLS handshake, Server hello (2):
* (304) (IN), TLS handshake, Unknown (8):
* (304) (IN), TLS handshake, Certificate (11):
* (304) (IN), TLS handshake, CERT verify (15):
* (304) (IN), TLS handshake, Finished (20):
* (304) (OUT), TLS handshake, Finished (20):
* SSL connection using TLSv1.3 / AEAD-CHACHA20-POLY1305-SHA256
* ALPN: server accepted h2
* Server certificate:
*  subject: CN=tiles.bbnavi.de
*  start date: Jan  9 11:13:07 2024 GMT
*  expire date: Apr  8 11:13:06 2024 GMT
*  subjectAltName: host "tiles.bbnavi.de" matched cert's "tiles.bbnavi.de"
*  issuer: C=US; O=Let's Encrypt; CN=R3
*  SSL certificate verify ok.
* using HTTP/2
* [HTTP/2] [1] OPENED stream for https://tiles.bbnavi.de/poi_coords_bike_repair_shops/17/70105/43050
* [HTTP/2] [1] [:method: GET]
* [HTTP/2] [1] [:scheme: https]
* [HTTP/2] [1] [:authority: tiles.bbnavi.de]
* [HTTP/2] [1] [:path: /poi_coords_bike_repair_shops/17/70105/43050]
* [HTTP/2] [1] [user-agent: curl/8.4.0]
* [HTTP/2] [1] [accept: */*]
> GET /poi_coords_bike_repair_shops/17/70105/43050 HTTP/2
> Host: tiles.bbnavi.de
> User-Agent: curl/8.4.0
> Accept: */*
> 
< HTTP/2 204 
< date: Wed, 17 Jan 2024 08:51:27 GMT
< vary: Origin, Access-Control-Request-Method, Access-Control-Request-Headers
< 
* Connection #0 to host tiles.bbnavi.de left intact
```

**Reply by epam-vkerkhoff** [reply, marked as answer]
Hi, 

it does indeed work. I checked a tile where I know for a fact that there is data with QGIS and everything is fine, the data points - where there is data - are shown. 

That means I need to change the URL that Digitransit-UI uses to call Martin, i.e. remove the .pbf at the end and the .public prefix. 

Thanks for all the help and assistance! Martin rocks!

**Comment by epam-vkerkhoff** [comment]
Issue solved with great help here and some investigation and testing from my side!

**Reply by nyurik** [reply]
sure thing, glad it worked out!
